requirements/darwin.txt

```diff
@@ -15,7 +15,8 @@ pycparser>=2.21
 pyopenssl>=19.0.0
 python-dateutil>=2.8.0
 python-gnupg>=0.4.4
-setproctitle>=1.1.10
+setproctitle>=1.1.10; python_version < '3.10'
```

Comment:
This issue looks to affect all platforms not just darwin. Can you update for all other OSs as well?

Reply:
All changes done!